.travis.yml

```diff
@@ -133,8 +133,9 @@ install:
     - source ci-helpers/travis/setup_conda.sh
 
 before_script:
-    # Download the sample data for the build of the documentation.
-    - if [[ $SETUP_CMD != *egg_info* ]]; then python -m "sunpy.data.sample"; fi
+    # Download the sample data for the online builds only.
+    - if [[ $SETUP_CMD == *online-only* ]]; then python -m "sunpy.data.sample"; fi
+    - if [[ $SETUP_CMD == *remote-data* ]]; then python -m "sunpy.data.sample"; fi
```

Comment:
If I could remember how to bash better these could probably be combined using some kind of or statement :laughing:

Reply:
I could try

Reply:
true.

Reply:
``if [[ $SETUP_CMD == *online-only* ]] || [[ $SETUP_CMD == *remote-data* ]]; then python -m "sunpy.data.sample"; fi``